pkg/codegen/python/gen.go

```diff
@@ -1527,15 +1527,14 @@ func (mod *modContext) genProperties(w io.Writer, properties []*schema.Property,
 		} else {
 			fmt.Fprintf(w, "%s    @pulumi.getter(name=%q)\n", indent, prop.Name)
 		}
+		if prop.DeprecationMessage != "" {
+			escaped := strings.ReplaceAll(prop.DeprecationMessage, `"`, `\"`)
+			fmt.Fprintf(w, "%s    @pulumi.deprecated(\"\"\"%s\"\"\")\n", indent, escaped)
```

Comment:
This means the generated SDK is now going to require a version of the `pulumi` PyPI package that has the `deprecated` decorator available, so we'll need to update the default min version range we emit and/or emit an "internal" utility decorator in the generated SDK itself (that sets `_pulumi_deprecated_callable`).